Fix integration tests after backend deps update

### Describe the bug

```
=== RUN   TestRgPystatsIntegration
    redis-gears_integration_test.go:23: 
                Error Trace:    redis-gears_integration_test.go:23
                Error:          "[]" should have 1 item(s), but has 0
                Test:           TestRgPystatsIntegration
--- FAIL: TestRgPystatsIntegration (10.00s)
=== RUN   TestRgDumpregistrationsIntegration
--- FAIL: TestRgDumpregistrationsIntegration (10.01s)
panic: runtime error: index out of range [0] with length 0 [recovered]
        panic: runtime error: index out of range [0] with length 0

goroutine 28 [running]:
testing.tRunner.func1.2(0x101623b40, 0x14000242480)
        /usr/local/go/src/testing/testing.go:1144 +0x25c
testing.tRunner.func1(0x14000182d80)
        /usr/local/go/src/testing/testing.go:1147 +0x384
panic(0x101623b40, 0x14000242480)
        /usr/local/go/src/runtime/panic.go:965 +0x14c
github.com/redisgrafana/grafana-redis-datasource/pkg.TestRgDumpregistrationsIntegration(0x14000182d80)
        /Users/mikhail/Documents/GitHub/grafana-redis-datasource/pkg/redis-gears_integration_test.go:43 +0x8b0
testing.tRunner(0x14000182d80, 0x1016791e8)
        /usr/local/go/src/testing/testing.go:1194 +0xe0
created by testing.(*T).Run
        /usr/local/go/src/testing/testing.go:1239 +0x21c
FAIL    github.com/redisgrafana/grafana-redis-datasource/pkg    20.249s
FAIL
Stopping redis-test ... done
Removing redis-test ... done
```

### Version
master

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -1,6 +1,6 @@
 name: CI
 
-on: [push, pull_request]
+on: [pull_request]
   
 jobs:
   build:

--- a/.github/workflows/integration.yml
+++ b/.github/workflows/integration.yml
@@ -1,8 +1,9 @@
 name: CI
 
-on: [pull_request]
+on:
+  pull_request
 
-  workflow_dispatch:
+  workflow_dispatch
   
 jobs:
   build:

--- a/.github/workflows/integration.yml
+++ b/.github/workflows/integration.yml
@@ -3,7 +3,7 @@ name: CI
 on:
   pull_request
 
-  workflow_dispatch
+  workflow_dispatch:
   
 jobs:
   build:

--- a/.github/workflows/integration.yml
+++ b/.github/workflows/integration.yml
@@ -1,0 +1,40 @@
+name: CI
+
+on: [pull_request]
+
+  workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: "12.x"
+
+      - name: Setup Go environment
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.14"
+
+      - name: Test backend
+        uses: magefile/mage-action@v1
+        with:
+          version: latest
+          args: cover
+
+      - name: Build backend
+        uses: magefile/mage-action@v1
+        with:
+          version: latest
+          args: buildAll
+
+      - name: Run integration tests
+        uses: magefile/mage-action@v1
+        with:
+          version: latest
+          args: integration

--- a/.github/workflows/integration.yml
+++ b/.github/workflows/integration.yml
@@ -1,9 +1,6 @@
 name: CI
 
-on:
-  pull_request
-
-  workflow_dispatch:
+on: [pull_request]
   
 jobs:
   build:

